### ISSUE TITLE
libexiv2.dylib missing in PyPI package

### ISSUE THREAD

**Issue by jsainio** [LeoHsiao1/pyexiv2#23]
Hi,

I tried using `pyexiv2` recently in a `conda` environment with Python 3.7.4 on macOS 10.15.4, and it seems there is a `libexiv2.dylib` missing from the package.

```
(base) jusic@jussis-mbp  % pip install pyexiv2
Collecting pyexiv2
  Downloading https://files.pythonhosted.org/packages/87/ff/9d8b88a731b463b7ff4e6b10c83069ff131ab359ec9c7491bb63c70a55a9/pyexiv2-2.3.0-py3-none-any.whl (3.1MB)
     |████████████████████████████████| 3.1MB 6.0MB/s
Installing collected packages: pyexiv2
Successfully installed pyexiv2-2.3.0
(base) jusic@jussis-mbp % python
Python 3.7.4 (default, Aug 13 2019, 15:17:50)
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyexiv2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jusic/anaconda3/lib/python3.7/site-packages/pyexiv2/__init__.py", line 6, in <module>
    from .core import *
  File "/Users/jusic/anaconda3/lib/python3.7/site-packages/pyexiv2/core.py", line 2, in <module>
    from .lib import exiv2api
  File "/Users/jusic/anaconda3/lib/python3.7/site-packages/pyexiv2/lib/__init__.py", line 23, in <module>
    ctypes.CDLL(os.path.join(lib_dir, 'libexiv2.dylib'))
  File "/Users/jusic/anaconda3/lib/python3.7/ctypes/__init__.py", line 364, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(/Users/jusic/anaconda3/lib/python3.7/site-packages/pyexiv2/lib/libexiv2.dylib, 6): image not found
```

If I separately `brew install pyexiv2` and then copy the dylib manually, it at least loads properly:
```
(base) jusic@jussis-mbp % cp /usr/local/Cellar/exiv2/0.27.3/lib/libexiv2*.dylib /Users/jusic/anaconda3/lib/python3.7/site-packages/pyexiv2/lib/
(base) jusic@jussis-mbp % python
Python 3.7.4 (default, Aug 13 2019, 15:17:50)
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyexiv2
>>>
```

Would it be possible to get the `dylib` packaged as well in the PyPI package?

**Comment by LeoHsiao1** [comment]
Oh, this is because I forgot to upload the dylib file. Thank you for reminding me.

**Comment by LeoHsiao1** [comment]
I have released [v2.3.1](https://github.com/LeoHsiao1/pyexiv2/releases/tag/v2.3.1), please download it.

**Comment by jsainio** [comment]
It works, thank you! That was fast. :)
